utils/build/docker/golang/parametric/datadog.go

```diff
@@ -101,6 +107,80 @@ func (s *apmClientServer) SpanSetError(ctx context.Context, args *SpanSetErrorAr
 	return &SpanSetErrorReturn{}, nil
 }
 
+type CustomLogger struct {
+	*logrus.Logger
+	globalConfig map[string]string
+}
+
+type Config struct {
+	Service                string            `json:"service"`
+	SampleRate             string            `json:"sample_rate"`
+	RuntimeMetricsEnabled  bool              `json:"runtime_metrics_enabled"`
+	Tags                   map[string]string `json:"tags"`
+	PropagationStyleInject string            `json:"propagation_style_inject"`
+	Debug                  bool              `json:"debug"`
+	Env                    string            `json:"env"`
+	DdVersion              string            `json:"dd_version"`
+}
+
+func (l *CustomLogger) Log(logMessage string) {
```

Comment:
Can you add a function description above line 126? Since this `Log()` method doesn't just "log," it does a lot more